spec/operations/mailers/user_added_to_project.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe Mailers::UserAddedToProject do
```

Comment:
As these aren't used, I'm trying to figure out how this operation specs aren't running / failing...